[2025-05-05T14:04:42.823+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-05T14:04:42.840+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-05-05T14:04:41.840682+00:00 [queued]>
[2025-05-05T14:04:42.848+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-05-05T14:04:41.840682+00:00 [queued]>
[2025-05-05T14:04:42.849+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-05-05T14:04:42.863+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-05-05 14:04:41.840682+00:00
[2025-05-05T14:04:42.869+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=718) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-05T14:04:42.870+0000] {standard_task_runner.py:72} INFO - Started process 751 to run task
[2025-05-05T14:04:42.871+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-05-05T14:04:41.840682+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmp557ao52d']
[2025-05-05T14:04:42.872+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask extract_raw_data
[2025-05-05T14:04:42.922+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-05-05T14:04:41.840682+00:00 [running]> on host 433e63c87a30
[2025-05-05T14:04:43.001+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-05T14:04:41.840682+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-05T14:04:41.840682+00:00'
[2025-05-05T14:04:43.002+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-05T14:04:43.016+0000] {lead_data_pipeline.py:189} INFO - Started batch processing for 2025-05-05.
[2025-05-05T14:04:43.017+0000] {lead_data_pipeline.py:33} INFO - Created synthetic data for RO country code.
[2025-05-05T14:05:11.480+0000] {lead_data_pipeline.py:100} INFO - Written 100372 records to the CSV file.
[2025-05-05T14:05:12.058+0000] {lead_data_pipeline.py:158} INFO - Added/Updated UUIDs in the dataset.
[2025-05-05T14:05:12.059+0000] {lead_data_pipeline.py:197} INFO - Finished batch processing 2025-05-05.
[2025-05-05T14:05:12.059+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-05T14:05:12.070+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-05T14:05:12.071+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-05-05T14:04:41.840682+00:00, execution_date=20250505T140441, start_date=20250505T140442, end_date=20250505T140512
[2025-05-05T14:05:12.105+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-05T14:05:12.128+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-05T14:05:12.130+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
